assets/css/detours/_detour_map.scss

```diff
@@ -26,7 +26,7 @@
 }
 
 .c-detour_map--detour-route-shape {
-  stroke: $color-lemon-500;
+  stroke: $color-eggplant-400;
```

Comment:
**suggestion:** I think if you add `stroke-opacity: 0.5` it will look even more like the hi-fi designs, as it is it still looks a bit different from the hi-fi designs for me.

Reply:
Good catch - thanks!

Reply:
[this should fix it](https://github.com/mbta/skate/pull/2458/commits/c15d0cf32c43da0cfc778f7ca0192f738f7bb00c)